src/Query/Builder.php

```diff
@@ -1004,8 +1004,14 @@ protected function compileWheres(): array
                 $where['boolean'] = 'or'.(str_ends_with($where['boolean'], 'not') ? ' not' : '');
             }
 
+            // Column name can be a Stringable object.
+            if (isset($where['column']) && $where['column'] instanceof \Stringable) {
```

Comment:
Why the check for `Stringable` here but not in other places? Is there a reason why we can't always cast to `string`?

Reply:
I needed a `if` statement to check if `column` is set, so I added the `instanceof` to avoid reassigning if the value is already a string. In other places, we know `$column` is set and we are not assigning the casted value.